config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
+      [
+         {
+           "apiVersion": "updateservice.operator.openshift.io/v1",
+           "kind": "UpdateService",
+           "metadata": {
+             "name": "update-service-instance"
+           },
+           "spec": {
+             "replicas": 1,
+             "releases": "openshift-release-dev/ocp-release",
+             "graphDataImage": "quay.io/cincinnati/demo-graph-data:for-testing-only"
+           }
+         }
+       ]
+    capabilities: Basic Install
+    description: Creates and maintains an OpenShift Container Platform Update Service instance
+    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: update-service-operator.v1.0.0
```

Comment:
I prefer [`openshift-update-service.vX.Y.Z`][1], to get closer to our 'OpenShift Update Service' official name.  No need to set an explicit version, some bundle-building magic will replace the `X.Y.Z` placeholder at build time.

[1]: https://github.com/openshift/cincinnati-operator/pull/79/files#diff-d31bf97881e5efe5323e57c437e9999604ff7d312eef59930c415444b3e7c66aR10